### DISCUSSION TITLE
ldap/openid/saml ... ?

### DISCUSSION THREAD

**Discussion by GPConnect** [simple-login/app#720] [Feature Requests]
Hello, 
i know simplelogin got his own single sign on implementation but i was wondering if we could expect other protocol added to connect user ? 
Thanks

**Comment by nguyenkims** [comment]
@GPConnect currently SimpleLogin is an OAuth2 IDP. By other protocols, do you mean SimpleLogin should become a SAML IDP for example?

**Reply by GPConnect** [reply]
i was talking more about being able to use our own provider (like keycloak for example) to log users to the dashboard !

**Reply by nguyenkims** [reply]
I see. Currently SimpleLogin supports "Login with Facebook/Google/GitHub" which is based on OAuth2 protocol so you can also adapt this to a generic keycloak IDP.

**Reply by muhlba91** [reply]
@GPConnect did you solve this in the end? i would like to use my own oauth/oidc provider as well but wasn't able to find a way to specify all required data like the oidc idp url and so forth.

@nguyenkims also login with facebook/google/github is now deprecated.i do understand the reason for this but wonder if you would add support for a custom oidc idp provider instead? in the end it's the users decision if they want to use a self-hosted idp or github/google/... or the built-in user auth or proton. but i think having the choice is important.
